client/gulpfile.js

```diff
@@ -33,3 +35,31 @@ gulp.task('build', function (callback) {
   runSequence('clean','copy_language_server','build_extension',callback);
 })
 
+gulp.task('bump', function () {
```

Comment:
I'm not sure we should be worrying about this right now, as publishing to the gallery has a similar function. I guess this doesn't hurt to have